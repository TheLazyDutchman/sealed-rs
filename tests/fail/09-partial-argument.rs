use sealed::sealed;

#[sealed(partial)]
pub trait A {
	#[seal]
	fn sealed() {}
}

#[sealed(partial)]
pub trait B {
	fn not_sealed();
}

fn main() {}
