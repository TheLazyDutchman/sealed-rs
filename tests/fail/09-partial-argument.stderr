error: This trait is partially sealed, however none of its methods are implementable, so the `partial` argument does nothing.
 --> tests/fail/09-partial-argument.rs:4:11
  |
4 | pub trait A {
  |           ^

error: This trait is partially sealed, however none of its methods are sealed, so it does nothing. Either seal a function or remove `partial` from the `sealed` attribute.
  --> tests/fail/09-partial-argument.rs:10:11
   |
10 | pub trait B {
   |           ^
